spec/actions_specs/reset_simulator_spec.rb

```diff
@@ -0,0 +1,11 @@
+describe Fastlane do
+  describe Fastlane::FastFile do
+    it "raises an exception when no device name is specified" do
+      expect do
+        Fastlane::FastFile.new.parse("lane :test do
+          reset_simulator({})
+        end").runner.execute(:test)
+      end.to raise_error
```

Comment:
It raises quite an exception here:
![image](https://cloud.githubusercontent.com/assets/350654/12863399/38a0ad7e-ccc9-11e5-864d-60a6d3b69d55.png)

If I get it right, to raise my own kind of exception, I have to add `verify_block` in action's `self.avilable_options`. Is it correct?

```
FastlaneCore::ConfigItem.new(key: :device,
                                       env_name: "FL_RESET_SIMULATOR_DEVICE",
                                       description: "Simulator name, UDID or \"all\""
                                       verify_block: proc do |value|
                                         raise "My custom message".red unless value and !value.empty?
                                       end)
```

Then I can 

```
end.to raise_error("My custom message")
```